### DISCUSSION TITLE
Webradio favorites & browsing sorting

### DISCUSSION THREAD

**Discussion by tsunulukai** [jcorporation/myMPD#665] [Ideas]
When browsing the webradio favorites, by default the webradios appears in a seeminly random order.
Having the possibility to either sort them by name would make navigation through favorites a lot easier.
Even better if we could arrange the tiles as wished (like we can do on the home page).

![image](https://user-images.githubusercontent.com/4215150/150846106-7bdbade4-7deb-4c2f-a29f-278612da52a1.png)

When browsing webradioDB and sorting webradios by country,language or genre, it would be nice if the entries could also be systematically sorted by name after the main sorting criteria.

![image](https://user-images.githubusercontent.com/4215150/150847082-89c44cff-abee-4402-a778-a603d3997d66.png)

**Comment by jcorporation** [comment]
Good points. I will implement secondary sort by Name for the WebradioDB view and sort by name for the favorites.

#666

**Reply by jcorporation** [reply]
Is now implemented, thanks for testing the devel version!

**Reply by tsunulukai** [reply]
My pleasure !
